test/native/setup.js

```diff
@@ -4,6 +4,22 @@
 import 'react-native-gesture-handler/jestSetup';
 import { Image, NativeModules as RNNativeModules } from 'react-native';
 
+// React Native sets up a global navigator, but that is not executed in the
+// testing environment: https://git.io/JSSBg
+global.navigator = global.navigator ?? {};
+
+// Set up the app runtime globals for the test environment, which includes
+// modifying the above `global.navigator`
+require( '../../packages/react-native-editor/src/globals' );
+
+if ( ! global.window.matchMedia ) {
+	global.window.matchMedia = () => ( {
+		matches: false,
+		addListener: () => {},
+		removeListener: () => {},
+	} );
+}
```

Comment:
Yes, I merely relocated [the original code](https://github.com/WordPress/gutenberg/pull/37715/files/487964747655863fe37866e8710cc6b3029a5bb0#diff-a51d3720ffdc1c754dd58fb5830a62174bb52392ca6655bf76f93d77cce61e02L124-L130) so that al mutations to `global` were colocated. Let me know if you'd prefer we revert this or locate them elsewhere, though.

Reply:
Oh no, I think the relocation is ok, I wouldn't revert it. My doubt was about the fact that this block is already located in the globals file, so I was wondering if it's really necessary to have it duplicated in the Jest setup as now we import the globals.

https://github.com/WordPress/gutenberg/blob/3d817f4b25f8f4748a2c81b0d55fa9a6dd73a350/test/native/setup.js#L11-L13

https://github.com/WordPress/gutenberg/blob/96836fe3905ac4c7d962b322b69fe88fc003eeed/packages/react-native-editor/src/globals.js#L54-L60

https://github.com/WordPress/gutenberg/blob/96836fe3905ac4c7d962b322b69fe88fc003eeed/test/native/setup.js#L127-L133

Reply:
Ah, I see. Thanks for noting this. I will remove the duplicative `matchMedia` definition from the test environment setup file.